README.md

```diff
@@ -111,10 +111,24 @@ $ cdk bootstrap
 $ cdk deploy                    # deploy -- this is a long process that will take about 40 minutes
 ```
 Other helpful CDK commands:
-
 * `cdk synth` outputs the CloudFormation configuration your CDK code produces
 * `cdk diff` outputs what infrastructure changes would be made by executing this deployment
 
+Troubleshooting CDK stuff:
```

Comment:
I'd probably just include all of these along with the "Other helpful CDK commands".  But also wonder if most of this is more appropriate for the knowledge-base.  You said you referred back here when starting with the cdk-api-playground example, which made me think this would have more visibility over there.

Reply:
Alright I'm cool with that